src/views/template/factory/factory.html

```diff
@@ -32,5 +32,16 @@
                 {{#l}}Place order{{/l}}
             </button>
         </form>
+        <form name="frequencyTrucks" action="">
```

Comment:
This part defines two forms (should be only one).

I think a button group may be better, since we also need an "increase capacity" button or similar. This would make that sufficiently generalisable. Then, bind to the click event on the button, like `$("#buy-truck").click(function (e) {...});` etc.
```html
<div class="btn-group" role="group">
    <button id="buy-truck">Buy truck</button>
</div>
```